pyfstat/make_sfts.py

```diff
@@ -584,7 +589,13 @@ def run_makefakedata(self):
         check_ok = self.check_cached_data_okay_to_use(cl_mfd)
         if check_ok is False:
             helper_functions.run_commandline(cl_mfd)
-            logging.info("Successfully wrote SFTs to: {}".format(self.sftfilepath))
+            if os.path.isfile(self.sftfilepath):
+                logging.info(f"Successfully wrote SFTs to: {self.sftfilepath}")
+            else:
+                logging.warning(
```

Comment:
This is bad: You may think you are doing stuff on a certain set of SFTs, but maybe you just made a wrong pattern and you are using everything but your actual SFTs. I'd kill the program if this fails.
```suggestion
                logging.critical(
```

Reply:
I was thinking "MFD has exited successfully, so technically the Writer has succeeded. We warn the user they didn't get back what they thought they would, and then they can decide to abort."

But of course there isn't any real exit status logic here, and manually checking log output isn't an option if the whole workflow is coded up as a script, like in our exampels. So probably you're right. If users want to continue after this, they can still do "try/except" and manually check if any sort of SFT file has been produced.

Reply:
A: "Hi, I'd like some water"
B: "Sorry, water machine broke, here's an empty glass"
A: "Thank you, this will quench my thirst".
Then A dies from dehydration.

Reply:
For the sake of an argument, I think in that example the machine wasn't obviously broken, the glass wasn't empty but contained whiskey instead of water, it had a matching "40% alc" label on it, and A only got a funny feeling because they didn't check the label before chugging it.

But in fact you still win this, because turning the warning into an error exposed a failing test and missing multi-IFO-pattern support. Fixed now.